### ISSUE TITLE
Replace carousel content on the homepage with scrollable content  

### ISSUE THREAD

**Issue by geekSiddharth** [openworm/org.openworm.website#116]
Carousel contains pretty important introductory content and links to important pages. People usually don't go through all the slides in a carousel. Replacing it with a scrollable content will help in increasing the visibility of that content.

**Comment by notsambeck** [comment]
I think this is a good idea!

I believe the goal should be to communicate more-or-less the same information that is currently in the carousel (brief descriptions of sub-projects), but in a way that is digestible very quickly.

Scrollable, or maybe a of grid view of pieces of the project? If the home page gets super long, it may end up having the same problem, that no one gets all the way to the end.

**Comment by geekSiddharth** [comment]
Yup. Grid view would be great. A few ideas for the grid structure:
- https://www.topcoder.com/getting-started/   (front side will have the title and the tag line of the project. back side will have explanations )
- One project per line. Each line would have two columns. One column would have visual data(images or video) and the other one would have textual details. (will make homepage a bit long)
 
Will add a few more ideas.

Yes, the refactoring thing is important.
